config.go

```diff
@@ -579,3 +579,29 @@ func (b *Builder) SetDomainname(name string) {
 func (b *Builder) SetDefaultMountsFilePath(path string) {
 	b.DefaultMountsFilePath = path
 }
+
+// Comment returns the comment which will be set in the container and in
+//containers built using images built from the container
+func (b *Builder) Comment() string {
+	return b.Docker.Comment
+}
+
+// SetComment sets the Comment which will be set in the container and in
+// containers built using images built from the container.
+func (b *Builder) SetComment(comment string) {
+	b.Docker.Comment = comment
+	b.OCIv1.History[0].Comment = comment
```

Comment:
The `History` list can be empty if the image is being built from scratch, and when it isn't empty, this is the entry for the base layer.

Reply:
Adding check for len ... I think adding the comment to the base layer is OK; at least that is what I have seen being done before.  I'll suggest a work-around in my next push.  or is there an alternate way you would want the comment set for v1?